<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestProprties</class>
 <widget class="QDialog" name="TestProprties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QScrollArea" name="right_scroll">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>350</width>
     <height>490</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>348</width>
      <height>488</height>
     </rect>
    </property>
    <widget class="QListWidget" name="h_pages_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>350</width>
       <height>490</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>590</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>הבא</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="left_scroll">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>350</width>
     <height>490</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>348</width>
      <height>488</height>
     </rect>
    </property>
    <widget class="QListWidget" name="e_pages_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>350</width>
       <height>490</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>9</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>דפים לבדיקה </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_all_epages">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>בחר הכל</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_all_hpages">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
    </font>
   </property>
   <property name="text">
    <string>בחר הכל</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
